src/navbar/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Menu }  from 'antd';
+import 'antd/dist/antd.css';
+import './style.css'
+
+
+class Navbar extends React.Component {
+  state={
+      current:null,
+        }
```

Comment:
Intenta mantener la indentación ordenada a medida que escribes código. De esa manera es más fácil ver cuándo empieza y cuándo termina una instrucción:
```suggestion
  state={
    current:null,
  }
```